File: lana/src/main/scala/com/financialforce/lana/cli/sfdx/logs/GetLogFile.scala
/*
 * Copyright (c) 2020 FinancialForce.com, inc. All rights reserved.
 */
package com.financialforce.lana.cli.sfdx.logs

import com.financialforce.lana.cli.sfdx.SFDX

import scala.concurrent.Future

object GetLogFile {
  def apply(path: String, logId: String): Future[String] =
    SFDX(path, Seq("force:apex:log:get", "--logid", logId))
}

File: ext/src/sfdx/logs/GetLogFile.ts
/*
 * Copyright (c) 2020 FinancialForce.com, inc. All rights reserved.
 */

import { SFDX } from "../sfdx";

export class GetLogFile {
  static async apply(path: string, logId: string): Promise<string> {
    return SFDX.apply(path, ["force:apex:log:get", "--logid", logId]);
  }
}
